tests/components/utility_meter/test_sensor.py

```diff
@@ -159,6 +165,71 @@ async def test_state(hass):
     assert state.state == "0.123"
 
 
+async def test_device_class(hass):
+    """Test utility device_class."""
+    config = {
+        "utility_meter": {
+            "energy_meter": {
+                "source": "sensor.energy",
+            },
+            "gas_meter": {
+                "source": "sensor.gas",
+            },
+        }
+    }
+
+    assert await async_setup_component(hass, DOMAIN, config)
+    assert await async_setup_component(hass, SENSOR_DOMAIN, config)
```

Comment:
These sensor integration will be loaded when loading the utility meter sensor platform.

Reply:
Fixed in ef5757a